charts/victoria-metrics-operator/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Next release
 
-- TODO
+- Moved crds to a shared chart and import them as a dependency
+- replaced `crd.*` properties to `crds.*` as they now belong to a subchart
```

Comment:
Please mention this as a upgrade note too, as it requires user to change their values if they used those properties before.